src/core/reducer.js

```diff
@@ -44,6 +44,12 @@ export function createGameReducer({ game, numPlayers, multiplayer }) {
   };
 
   const state = game.flow.init({ G: initial.G, ctx: initial.ctx });
+
+  // either take the provided seed, or use seedrandom to create one.
+  // Math.seedrandom returns an autoseed with local entropy only.
+  const seed = game.seed === undefined ? Math.seedrandom() : game.seed;
```

Comment:
I personally think it's better to pass `seed` down to `FlowWithPhases` in `game.js` and have it initialize `ctx` properly.